en/docs/references/adaptive-authentication-js-api-reference.md

```diff
@@ -607,35 +606,35 @@ step number.
     Nginx) with connection termination, the ip is retrieved from the
     headers set by the load balancer.
 
-##### response Object
+#### response object
 
 -   `          response.headers[“<header_name>”]         ` : (Write)
     Response header value for the given header name by
     &lt;header\_name&gt;
 
-##### session Object
+#### session object
 
 -   `          session.userAgent         ` :  This is userAgent object of the user session.. See [userAgent
-    Object](#userAgent-Object) for more
+    object](#useragent-object) for more
     information.
 -   `          session.ip         ` :  This is the session’s IP address.
 -   `          session.loginTime         ` :  This is the session’s last login time.
 -   `          session.lastAccessTime         ` :  This is the session’s last accessed time.
 -   `          session.id         ` :  This is the session’s id.
 -   `          session.applications         ` :  This is the list of application objects in the session. See [application
-    Object](#application-object) for more
+    object](#application-object) for more
     information.
 
-##### application Object
+#### application object
 
 -   `          application.subject         ` :  This is the subject of the application.
 -   `          application.appName         ` :  This is the name of the application.
 -   `          application.appId         ` :  This is the id of the application.
     
-##### userAgent Object
+#### userAgent object
 
 -   `          userAgent.raw         ` :  This is the raw userAgent string.
--   `          userAgent.browser         ` :  This is the Web Browser property that is extracted from the raw userAgent string.
--   `          userAgent.platform         ` :  This is the Operating System property that is extracted from the raw userAgent string.
--   `          userAgent.device         ` :  This is the Device property that is extracted from the raw userAgent string.
+-   `          userAgent.browser         ` :  This is the web browser property that is extracted from the raw userAgent string.
```

Comment:
Shall we give placeholders like userAgent as codes? 
The same applies to 'true' and 'false'. 

Please check wherever applicable.

Reply:
fixed in https://github.com/wso2/docs-is/pull/2617